### ISSUE TITLE
OpenLive for any interface in MacOS

### ISSUE THREAD

**Issue by tjjh89017** [packetcap/go-pcap#58]
Is it possible to have a OpenLive handel to listen all interfaces in once?
Thank you

Something like `tcpdump -i any`

**Comment by deitch** [comment]
It does it already. What happens if you pass `""` as the iface?

**Comment by tjjh89017** [comment]
I tired sample code with the following command `sudo ./pcap` (default is "")
I got this error.
```
capturing from interface
FATA[0000] failed to set the BPF interface: error: 6
```

**Comment by tjjh89017** [comment]
By the way, I just tested `tcpdump` in MacOS M3.
I cannot use `-i any`
I'm not sure if there is some way to have the same behavior like Linux for Raw Socket with SO_ATTACH_FILTER on all interfaces

**Comment by deitch** [comment]
Yeah, macOS doesn't have the same constructs (and the same way of working). The interface access call is [here](https://github.com/packetcap/go-pcap/blob/master/pcap_darwin.go#L137-L140). You would need to have an option to set a loop for it. Feel free to open a PR.

**Comment by tjjh89017** [comment]
@deitch Hi
I found the problem that I cannot setup BPF filter in the MacOS.
Please help me to review #60 
Thank you!
